first-year-scraper/generate_schedule.py

```diff
@@ -30,7 +30,7 @@ def generate_schedule(valid_sheets, workbook, subjects_dict):
             # manually deleting that column:
             worksheet.delete_cols(7,1)
         
-        worksheet.delete_cols(7,1)
+        worksheet.delete_cols(9,1)
```

Comment:
Why was this changed?

Reply:
Earlier it was creating a new .xlsx file from the original by deleting two irrelevant rows and columns each and then generate the schedule from the new file. Now it generates directly from the original, hence the change.